quickstarts/Function_calling_config.ipynb

```diff
@@ -48,323 +48,7 @@
-        "## Automatic function calling\n",
-        "\n",
-        "`tool_config` works when enabling automatic function calling too."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": 9,
-      "metadata": {
-        "id": "hx7aIX8OXvi6"
-      },
-      "outputs": [
-        {
-          "name": "stdout",
-          "output_type": "stream",
-          "text": [
-            "LIGHTBOT: Lights enabled.\n"
-          ]
-        },
-        {
-          "data": {
-            "text/plain": [
-              "response:\n",
-              "GenerateContentResponse(\n",
-              "    done=True,\n",
-              "    iterator=None,\n",
-              "    result=protos.GenerateContentResponse({\n",
-              "      \"candidates\": [\n",
-              "        {\n",
-              "          \"content\": {\n",
-              "            \"parts\": [\n",
-              "              {\n",
-              "                \"text\": \"Okay, I've turned on the lights.\"\n",
-              "              }\n",
-              "            ],\n",
-              "            \"role\": \"model\"\n",
-              "          },\n",
-              "          \"finish_reason\": \"STOP\",\n",
-              "          \"avg_logprobs\": -0.00836455523967743\n",
-              "        }\n",
-              "      ],\n",
-              "      \"usage_metadata\": {\n",
-              "        \"prompt_token_count\": 86,\n",
-              "        \"candidates_token_count\": 10,\n",
-              "        \"total_token_count\": 96\n",
-              "      },\n",
-              "      \"model_version\": \"gemini-2.0-flash\"\n",
-              "    }),\n",
-              ")"
-            ]
-          },
-          "execution_count": 9,
-          "metadata": {},
-          "output_type": "execute_result"
-        }
-      ],
-      "source": [
-        "available_fns = [\"enable_lights\"]\n",
-        "tool_config = tool_config_from_mode(\"any\", available_fns)\n",
-        "\n",
-        "auto_chat = model.start_chat(enable_automatic_function_calling=True)\n",
-        "auto_chat.send_message(\"It's awful dark in here...\", tool_config=tool_config)"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "kz8McBZfXg0N"
-      },
-      "source": [
-        "## Further reading\n",
-        "\n",
-        "Check out the function calling [quickstart](https://github.com/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb) for an introduction to function calling. You can find another fun function calling example [here](https://github.com/google-gemini/cookbook/blob/main/quickstarts/rest/Function_calling_REST.ipynb) using curl.\n"
+        "This notebook has been migrated to the new [`examples/function_calling.ipynb`](https://github.com/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb) notebook highlighting the latest Gemini capabilities and formatting."
```

Comment:
Can you use a relative link instead (`./Function_calling.ipynb`) so that it woks in IDE, in Colab, etc...?
Also, minor typo, but the function calling notebook is in the quickstarts folder :)